transport/sender.go

```diff
@@ -0,0 +1,28 @@
+package transport
+
+import (
+	"io"
+	"time"
+)
+
+type MultiSender struct {
+	Transports *Registry
+	OnError    func(Transport, error)
+}
+
+func (sender MultiSender) Send(buffer TimestampWriterTo) (int64, error) {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
exported method MultiSender.Send should have comment or be unexported